demo/index.html

```diff
@@ -12,12 +12,20 @@
 			};
 		};
 
-		var t = titlebar()
+		var tm = titlebar({'os':'mac'})
 			.appendTo(document.body)
 			.on('close', log('close'))
 			.on('minimize', log('minimize'))
 			.on('fullscreen', log('fullscreen'))
 			.on('maximize', log('maximize'));
+
+		document.write('<br>');
+
+		var tw = titlebar({'os':'win'})
+			.appendTo(document.body)
+			.on('close', log('close'))
+			.on('minimize', log('minimize'))
+			.on('maximize', log('maximize'));
```

Comment:
That's intentional. It's the behavior Windows 10 uses when maximizing, and I checked that the height is correct.